docs/topics/development/dependencies.md

```diff
@@ -52,6 +52,18 @@ This prevents pip from installing transitive dependencies.
 
 We do this because it gives us control over the full dependency chain - we know exactly which version of what package is installed so we can fully reproduce & trust environments.
 
+In order to determine the dependencies a given package requires you can check
+
+```bash
+pip show <package-name>
+```
+
+To see the `requirements` field which lists the dependencies. Install missing dependencies manually.
+
+```{admonition} Note
+Ensure to document in the requirements file which "main" dependency a "transitive" dependency is required by.
```

Comment:
```suggestion
Ensure to comment in the requirements file above transitive dependencies which direct dependency it is required by.
```
?

Reply:
Although I'm not sure how reliable manually adding the information is.

Reply:
We have been doing it until now (AFAICS) and probably better to have it than not.. ideally we'd have a pip.lock.py type file to do it for us, but alas.